magenta/protobuf/music.proto

```diff
@@ -271,6 +274,15 @@ message NoteSequence {
     string name = 2;
   }
 
+  // Stores information about an instrument name
+  // See usage within Note for more details.
+  message InstrumentInfo {
+    // The instrument index.
+    int32 inst_channel = 1;
```

Comment:
I would call this `instrument`, to match the name of the field in `Note`. Having "channel" in the name is potentially misleading because the instrument id does not necessarily map to the MIDI channel.

Reply:
I did that change and committed it. Sorry for the addition and deletion of commits. It is my first contribution to public GitHub and didn't find the way to delete commits (I deleted the files that by mistake ware added to the wrong place).